account_invoice_pricelist_sale/tests/test_module.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestModule(TransactionCase):
+    def setUp(self):
+        super(TestModule, self).setUp()
```

Comment:
Apply PEP 3135 (28-Apr-2007) to "super()."
```suggestion
        super().setUp()
```

Reply:
updated